app/services/concerns/request_performer.rb

```diff
@@ -13,7 +13,7 @@ module RequestPerformer # rubocop:disable Metrics/ModuleLength
   ].freeze
 
   # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
-  def perform_request(http_verb, url, options = {})
+  def perform_request(http_verb, url = 'localhost:3000', options = {})
```

Comment:
Este `localhost` no debería ser necesario, y además no correspondería porque la url debería ser la de la api externa, no la interna nuestra

```suggestion
  def perform_request(http_verb, url, options = {})
```